apps/rotmgfame/rotmg_fame.star

```diff
@@ -0,0 +1,94 @@
+"""
+Applet: ROTMG Fame
+Summary: Display your account fame
+Description: Displays your account fame and rank from realmeye.
+Author: d3or
+"""
+
+load("encoding/base64.star", "base64")
+load("http.star", "http")
+load("render.star", "render")
+load("schema.star", "schema")
+
+FAME_ICON = base64.decode("""
```

Comment:
Might be good to try to reduce the size of this graphic a bit, ifd you haven't already. Scaling down the file to an appropriate resolution before including it in the .star usually helps a lot. Another trick is to strip EXIF information, if present.